backend/src/config.ts

```diff
@@ -29,7 +29,9 @@ export const config = merge(
 
     intervals: {
       checkFinalityStatus: SECOND,
-      checkChainBlockStats: SECOND,
+      checkLatestBlockHeight: SECOND,
+      checkLatestGasPrice: 15 * SECOND,
+      checkBlockProductionSpeed: 5 * SECOND,
```

Comment:
@frol 
Please make a sanity check of these three intervals.
I guess gas price hardly ever updates.
Can we afford 5second lag in block production speed or should we be more frequent?

Reply:
These look reasonable to me